### PR TITLE
New package: libsignal-protocol-c-2.3.2

### PR DESCRIPTION
**Pull request by teldra**

### CODE REVIEW

File: srcpkgs/libsignal-protocol-c/template [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Template file for 'libsignal-protocol-c'
+pkgname=libsignal-protocol-c
+version=2.3.2
+revision=1
+build_style=cmake
+makedepends="check-devel lcov libressl-devel"
```
Comment (maxice8) [inline srcpkgs/libsignal-protocol-c/template]: isn't check-devel used only when building tests?

Comment (xtraeme) [reply srcpkgs/libsignal-protocol-c/template]: Yeah, it should.

Comment (teldra) [reply srcpkgs/libsignal-protocol-c/template]: I would like to integrate the check, if someone has a suggestion, how I do that.

Comment (teldra) [reply srcpkgs/libsignal-protocol-c/template]: I have left this for now.

**Comment by maxice8** [comment]
On Mon, Apr 15, 2019 at 02:44:29AM -0700, Teldra wrote:
> teldra commented on this pull request.
> 
> 
> 
> > @@ -0,0 +1,12 @@
> +# Template file for 'libsignal-protocol-c'
> +pkgname=libsignal-protocol-c
> +version=2.3.2
> +revision=1
> +build_style=cmake
> +makedepends="check-devel lcov libressl-devel"
> 
> I have left this for now.
> 
> -- 
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/10955#discussion_r275282853

guard it under $XBPS_CHECK_PKGS like we do with CROSS_BUILD

**Comment by teldra** [comment]
Done and should be ready to merge.

File: srcpkgs/libsignal-protocol-c/template [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+build_style=cmake
+makedepends="libressl-devel"
+checkdepends="pkg-config check-devel"
+short_desc="Signal Protocol C Library"
+maintainer="teldra <teldra@rotce.de>"
+license="GPL-3.0-or-later"
+homepage="https://www.whispersystems.org/"
+distfiles="https://github.com/signalapp/libsignal-protocol-c/archive/v${version}.tar.gz"
+checksum=f3826f3045352e14027611c95449bfcfe39bfd3d093d578c70f70eee0c85000d
+
```
Comment (maxice8) [inline srcpkgs/libsignal-protocol-c/template]: you can put -DBUILD_TESTING= behind a XBPS_CHECK_PKGS check

**Pull request merged**